modules/ROOT/pages/config-workflow.adoc

```diff
@@ -5,12 +5,16 @@ endif::[]
 
 . Setup a balancer as described in xref:install-create-lb.adoc[To Setup a Load Balancer].
 
-. Perform additional installation tasks as described in xref:install-add-tasks.adoc[To Perform Additional Installation Tasks].
-
-. Ensure that you have configured your system to perform regular backups. See xref:backup-and-disaster-recovery.adoc[About Backup and Recovery].
+. Ensure that you have configured your system to perform regular backups. See xref:backup-and-disaster-recovery.adoc[About Backup and Recovery]
 
-. Configure an external authentication provider. See xref:install-config-ldap-pce.adoc[To Configure LDAP in Anypoint Platform Private Cloud]
-
-. (Optional) Delete the default Anypoint Platform user created during installation. See xref:install-disable-local-user.adoc[To Delete the Local User after Installing Private Cloud Edition].
+. Perform additional installation tasks as described in xref:install-add-tasks.adoc[To Perform Additional Installation Tasks].
+    * xref:access-management-dns.adoc[Configure DNS]
+    * xref:access-management-security.adoc[Set Certificates]
+    * Log Forwarding
+    * xref:access-management-smtp.adoc[Configure SMTP]
```

Comment:
Same issue here: Our link checker is flagging this as a broken link. 
For this to work, you need to either rename the link to 

> See xref:access-management-SMTP.adoc[Configure SMTP].

Or rename the filename to use letter case.

Reply:
Done

Reply:
This particular link is still being flagged. I see you changed it for the `install-add-tasks.adoc` file, but it needs to be changed here as well.

Reply:
sorry, I thought I replaced all, Done.